.github/workflows/release.yaml

```diff
@@ -186,7 +202,9 @@ jobs:
           registry-url: "https://registry.npmjs.org"
 
       - name: Publish package on npm
-        run: npm publish
+        run:
+          npm publish ${{ github.event.inputs.dry-run == 'true' && '--dry-run'
```

Comment:
(but here it looks like `== 'true'` *is* required, since inside the braces, everything is a string...

Reply:
That's true. `== 'true'` seems needed here.

Reply:
OK — just to confirm, we're sure that `${{ !inputs.dry-run }}` will evaluate as `true` when `dry-run` is false?

To quote the [docs](https://docs.github.com/en/actions/learn-github-actions/expressions): 

> Using the ${{ }} expression syntax turns the contents into a string, and strings are truthy. For example, if: true && ${{ false }} will evaluate to true.

I had previously thought that `!inputs.dry-run` would evaluate as `true` when `dry-run` is `false`, because it's not inside the braces (but all of our are inside braces)